image/tarexport/save.go

```diff
@@ -36,8 +38,8 @@ type saveSession struct {
 	*tarexporter
 	outDir      string
 	images      map[image.ID]*imageDescriptor
-	savedLayers map[string]struct{}
-	diffIDPaths map[layer.DiffID]string // cache every diffID blob to avoid duplicates
+	savedLayers map[layer.DiffID]distribution.Descriptor
+	savedImages map[string]struct{}
```

Comment:
nit: `savedImages` is used in `saveConfig`, you might want to give these the same name

Reply:
Done